docs/tutorials/pre_executed/ztf-alerts-sne.ipynb

```diff
@@ -0,0 +1,2117 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fc1a0e0de461de1",
+   "metadata": {},
+   "source": [
+    "# Search for SN-like light curves in ZTF alerts\n",
+    "\n",
+    "We will use lsdb package to load a Hats catalog with [ZTF](https://www.ztf.caltech.edu) alerts.\n",
+    "The dataset contains all alerts sent from the beginning of the survey until 2023-09-13 corresponding to objects having at least 20 detections.\n",
```

Comment:
Do you know when was the beginning of the survey?

Reply:
I found the minimal MJD with LSDB =)